core/rawdb/ancient_utils.go

```diff
@@ -81,7 +81,8 @@ func inspectFreezers(db ethdb.Database) ([]freezerInfo, error) {
 			}
 			info.tail = tail
 			infos = append(infos, info)
-
+		case stateFreezerName:
```

Comment:
Why don't we implement it, rather than continue and without no any err prompt?

Reply:
And I noticed that the `go-ethereum` already implement it.  

https://github.com/ethereum/go-ethereum/blob/5c6f4b9f0d4270fcc56df681bf003e6a74f11a6b/core/rawdb/ancient_utils.go#L79C1-L116C1

Reply:
rebase the newest eth pbss commit again, include this part.